docs/implementation.adoc

```diff
@@ -1279,20 +1279,20 @@ The webhook receiver would start by parsing the payload and then create the appr
 JsonNode json = new ObjectMapper().readTree(payload);
 String event = json.path("event").asString();
 if ("branch-create".equals(event)) {
-    SCMHeadEvent.fireNow(new MyBrachSCMHeadEvent(Type.CREATED, json);
+    SCMHeadEvent.fireNow(new MyBranchSCMHeadEvent(Type.CREATED, json);
 } else if ("branch-update".equals(event)) {
-    SCMHeadEvent.fireNow(new MyBrachSCMHeadEvent(Type.UPDATED, json);
+    SCMHeadEvent.fireNow(new MyBranchSCMHeadEvent(Type.UPDATED, json);
 } // else etc
 ----
 
 Because each event from the source control system has a 1:1 correspondance with the events in the SCM API the implementation of each event can be fairly straightforward:
 
 [source,java]
 ----
-public class MyBrachSCMHeadEvent extends SCMHeadEvent<JsonNode> {
+public class MyBranchSCMHeadEvent extends SCMHeadEvent<JsonNode> {
```

Comment:
Isn't this breaking binary compatibility?  

Maybe more of a 🐜  than a 🐛  if these APIs are not published yet, but be careful here to make sure this is updated wherever needed.